.eslintrc

```diff
@@ -1,6 +1,8 @@
 {
   "extends": "airbnb",
   "rules": {
+    "default-case": 0,
```

Comment:
I think both this and next rule are generally good ideas; so I don't mind overriding them when an exception is needed. Does this map to something in the web app by chance?

Reply:
Nope, I'll add them back in now. Thanks for the heads up